src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -60,7 +65,7 @@ public StorageOperationException(String message) {
      * @throws InvalidStorageFilePathException if the default path is invalid
      */
     public StorageFile() throws InvalidStorageFilePathException {
-        this(DEFAULT_STORAGE_FILEPATH);
+        this(DEFAULT_STORAGE_FILEPATH); // this calls StorageFile(String filePath)
```

Comment:
Perhaps there is no need to update with an inline comment in this context?
If you feel that this comment is needed, would updating the method header comment be better?